### PR TITLE
Moves away missions to space ruins, removes gateway

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This PR:
Moves Centcomm away (russia gateway), Black market meat packers, UO71, and Moon outpost 19 from gateway maps to space ruins.
Removes checks and other things to do with away missions.
UO71 terrors now apply gateway transition logic that gibbed/prevented issues to entering the station z-level only.
UO71 now has some invulnerable rock walls to prevent cheesing the level.
Swarmers now spawn randomly using the nukedisk respawn landmark (could not think of a better option).
Ex-gateway rooms become "abandoned gateway" rooms, currently unpowered and used until I can think of a better option.

TODO:

- [ ] Figure out a better use for the old room
- [ ] Another way for swarmers to be spawned?
- [ ] Should we turn chill area of beach into space ruin? who would use it? consider.
- [ ] Thoroughly sanity check UO71 terror spider behaviour.
- [ ] Figure why ruin spawning causes smoothing errors.
- [ ] Shout at headmins to remove exile stuff

## Why It's Good For The Game
The Gateway has long been slated for attention, as determined in community meetings past. I thought it would be a shame to lose some of these levels so why not make them space ruins?

## Images of changes
image
image
image

## Changelog
:cl:
del: Deleted the gateway
tweak: Moved gateway missions (apart from beach) to be space ruins
/:cl:

### CODE REVIEW

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -293,3 +293,35 @@
...
+	allow_duplicates = FALSE
+	cost = 5
+
+/datum/map_template/ruin/space/terrorspiders
+	id = "terrorspiders"
+	suffix = "terrorspiders.dmm"
+	name = "Research Facility 265"
+	description = "The screams of long-dead staff can still be heard at times."
+	allow_duplicates = FALSE
+	cost = 5
```
Comment (AffectedArc07) [inline code/datums/ruins/space.dm:327]: Im *really* not sure on the feasibility of converting these ones to space without **major** cutdowns. The gateways themselves are huge. I also worry about the terror spiders one, theres a **LOT** of hardcoded z-level checks on them, including things like the wireless eguns.

If you lock those to the Z-level that the ruin spawns on, not only would you have to track that, but that carries its own implications. What if the depot spawns on the same level as this? What if someone brings a terror back to the station by some other means that dont rely on gateway transit?

File: code/defines/vox_sounds.dm [hunk 2]
```diff
@@ -473,7 +473,6 @@ GLOBAL_LIST_INIT(vox_sounds, list(
 "fuel" = 'sound/vox_fem/fuel.ogg',
 "g" = 'sound/vox_fem/g.ogg',
 "gas" = 'sound/vox_fem/gas.ogg',
-"gateway" = 'sound/vox_fem/gateway.ogg',
```
Comment (AffectedArc07) [inline code/defines/vox_sounds.dm:476]: We still have old words for things such as shadowlings and the like, keep this.

File: code/game/gamemodes/miniantags/bot_swarm/swarmer_event.dm [hunk 3]
```diff
@@ -5,18 +5,15 @@
 /datum/event/spawn_swarmer/announce()
 	if(prob(25)) //25% chance to announce it to the crew
 		var/swarmer_report = "<font size=3><b>NAS Trurl High-Priority Update</b></span>"
-		swarmer_report += "<br><br>Our long-range sensors have detected an odd signal emanating from your station's gateway. We recommend immediate investigation of your gateway, as something may have come \
+		swarmer_report += "<br><br>Our long-range sensors have detected an odd signal emanating from your station. We recommend immediate investigation, as something may have come \
```
Comment (AffectedArc07) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer_event.dm:8]: Come through from *where*?

File: code/game/turfs/simulated/floor/beach.dm [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/turf/simulated/floor/beach/away
```
Comment (AffectedArc07) [inline code/game/turfs/simulated/floor/beach.dm:1]: Still got away in the path

File: config/example/config.toml [hunk 5]
```diff
@@ -644,6 +621,10 @@ active_space_ruins = [
 	"_maps/map_files/RandomRuins/SpaceRuins/onehalf.dmm",
 	"_maps/map_files/RandomRuins/SpaceRuins/syndiecakesfactory.dmm",
 	"_maps/map_files/RandomRuins/SpaceRuins/wizardcrash.dmm",
+	"_maps/map_files/RandomRuins/SpaceRuins/centcomAway.dmm",
```
Comment (AffectedArc07) [inline config/example/config.toml:624]: This should really be lowercase but git doesnt like case renames so

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/miniantags/bot_swarm/swarmer_event.dm (hunk 3 above)
Comment (S34NW) [reply code/game/gamemodes/miniantags/bot_swarm/swarmer_event.dm:8]: yes

File: code/game/turfs/simulated/floor/beach.dm (hunk 4 above)
Comment (S34NW) [reply code/game/turfs/simulated/floor/beach.dm:1]: Yeah, I know its a PITA but this should be part of a wider water/beach turf refactor IMO, there's a lot of crap piled onto these and these turfs work in a really dumb way a lot of the time. Not to mention there is a non-away turf path for these that does something different... go figure.

**Comment by Norwes1** [comment]
Really? That'd be a shame if so. Space is a right shitshow to navigate, and without spacepods moving other people through it (dead bodies for recovery, prisoners/abductees, etc.) is downright painful. It's teleporters or nothin', which makes it effectively no different from the Gateway again.

Honestly, I don't see any reason to remove the Gateway at this time, doubly so since it makes for a great starting point for admin events. It's easy to host some off-station drama there, where people can choose to participate (or not), and is easily navigable for even the baldest of 'tiders.

**Comment by PacifistDalek** [comment]
It’s definitely worth holding onto them in some fashion but the limitations that space ruins have (inaccessibility, limited sets of gear to get there so you can’t have a lot of people come with, etc) are a bit sad as options too.

i wonder if something like gateway item/room could continue to exist as a special teleporter to those space runs that would only activate upon someone space travelling there and activating the far end. That way adminbus events or even just a gang of bored assistants could easily go hang out at the beach or whatnot could still easily happen, but it would still shift focus to space exploration?

IDK, i don’t know the reasons for gateway removal so this idea might conflict with that

**Comment by Ordosian** [comment]
As a very active Gateway Invader. Removing the gateway would not only remove a fun extra role for explorers but just completely remove a unique experience, like even the Gateways that don't offer much but some exploration can be really fun and a good way to pass time without having to navigate space. 

For example, the Terror Gateway provides a fun way to be introduced to Terror Spiders and drops some lore tidbits around with research reports. And the loot isn't even the reason to do it. Yes, the loot is beneficial to an agent, but otherwise, it is just fun for a normal player like myself to attempt it.

And yes, I do understand you wish to make them into space ruins but it just completely doesn't fit and like Norwest above has said. Space exploration is a shitshow, navigation is annoying and moving people through Z levels is even more annoying. The entire premise for Gateway missions is that it can only be accessed by the Gateway. It would be very weird to find a massive asteroid that spans basically the entire z level with terrors on it.

Does Gateway need some love? Yes, it 100% does but I don't believe removing it because it is in a limbo state is the best way to do it. 

And as Norwest has stated. Gateway offers the best potential for Events that are not forced. If players wish to participate, they can walk in. Like earlier, Exavere made the Terror Gateway harder but gave Anomaly cores as the end reward. Sent the fax to Command from CC. Nothing forced and sadly Command couldn't form a team due to on-station problems but players could still go in and attempt it.  And as the person that did attempt it. It was really fun. 

Also, note. If an explorer in space (Not station Z level) is infected with White Terror Eggs, those spiderlings can still traverse space. Spiderlings do not die in space, only Terrors themselves. But for example, if the Explorer was waiting outside the Station Z level for the eggs to clear, those spiderlings can most certainly enter the station Z level and cause chaos.

**Comment by MRgaty** [comment]
I'm worried about how exiling prisoners will work if the gateway gets removed.

**Comment by TrashDoxx** [comment]
The change must be reversed. Fuck space. Players should not be able to go anywhere other than the gateway, station & lavaland.

**Comment by Norwes1** [comment]
Actually, I think my prior comment was a little hesitant. I am dead-set against this change, and see removing the Gateway as another case of "remove without replacing:"

1. Education: Gateway missions provide an **excellent** place for wannabe explorers to learn how2hostile environment. There's risks, to be sure, but it's a much simpler and easier place to navigate than anything space-related, and still considerably less dangerous than Lavaland. Someone trying to explore space has to navigate the sheer hell of z-levels, and speaking as someone with *ahem* number of hours spent flying through SS13 space, it's still enough to make my head hurt at times. Gateway missions require no prior introduction, or understanding of how to know when you're looping in space: you hop through the stargate, and you're immediately dropped off into the action. That's a far better place for people to learn when they're still struggling with things like internals, rather than forcing them to learn how2navigate non-linear space merely to reach something interesting.

2. Variety loss: Adding Gateway levels to space, assuming that's even possible and feasible, will naturally come at the expense of everything already in space. The Gateway provides a z-level's worth of Stuff To Do, all without overly interfering with the main station or space exploration (or ghost roles, too). Losing the Gateway means fewer Theta Station opportunities, because it'll spawn considerably less often. Similarly, losing the Gateway means more problematic interactions between the Syndie researchers and crew, because bored Explorers will naturally go out and find their own entertainment if it isn't provided for them. Better to keep people entertained going out gunning down NPCs, then ending up in situations that might require a bwoink.

3. Lack of gain: The Gateway effectively costs nothing to the station in general. It is, at worst, a source of antag loot - but complaining about a chameleon projector from a Gateway mission is pretty absurd when there's Lavaland and its validhunting Miners riiiiiight over there. What is gained by cutting it? What do we actually get from taking out yet more content? If people hate swarmers, then just cut swarmer spawns. If there's a feeling that the Gateway is in limbo, then that's an obvious call to _add_ more to it. I frequently use the Gateway levels myself for exactly that, whether it's running a Stepford-esque bunch of terrified crew in the xeno Gateway, having strange chill-out sessions in the beach, having running three-way firefights between Explorers, spiders, and Syndicate agents, or major events like the STALKER-esque event I've run in the Russian one. Those sorts of things can't be easily replicated, because they rely on fast travel to and from the location. Lose the Gateway, and those moments are outski.

To be frank, this feels like another "remove without replacing" case like what happened to spacepods. They were cut for being bad code, which I won't argue with, but all the talk of "maybe adding something to replace them" came to nothing. Getting cargo, especially human cargo (bodies, prisoners, etc.) is still a nigh-impossible challenge today without spacepods or on-site teleporters, and all the talk of replacements has sputtered out. If we're going to lose the Gateway, I'd like to see a concrete plan to actually make space exploration viable, because that's all you're leaving behind. And let's not forget, space exploration as it stands today is an absolute mess, requiring advanced knowledge of strange in-game mechanics like z-level construction just to get someplace interesting.

If it causes significant server lag, fine, I'll shut my trap. But barring some immediate need like that, I just don't see the need to remove a functional and frequently interesting, engaging thing without a viable means of replacing it.

**Comment by S34NW** [comment]
Norwest I think some of your arguments are valid, especially the education one (at least when older players sdon't just rush it for the loot).

But I think you make some really bad takes and false claims to try and make your point:

> 2\. The Gateway provides a z-level's worth of Stuff To Do  ...  Losing the Gateway means fewer Theta Station opportunities, because it'll spawn considerably less often

This is just not true, gateway levels take up at MOST 1 quarter of a z-level. They are all about the size of the russian ringed space ruin. (The puzzle one). They just seem bigger because there is no frame of reference. So not only are we not "losing an entire z-level's worth of stuff to do" because of size, we also aren't losing it at all, since it's still hanging about. There are so many z-levels that get populated with ruins, the likelyhood of it spawning instead of x or y ruin is slim, a lot of space ruins are duplicated. Wild clames of "theta spawning considerable less often" is just alarmist rhetoric with no statistical evidence. Do not make claims like this without actually doing a controlled test.

> 2\. Better to keep people entertained going out gunning down NPCs, then ending up in situations that might require a bwoink.

This is just bonkers, players aren't going to go start breaking server rules because the gateway content requires some effort to run. This is more alarmist rhetoric with no basis in reality.

> 3\. but complaining about a chameleon projector from a Gateway mission is pretty absurd

Nobody is complaining about a single cham projector. People are greatly concerned about things like 6 free turret guns that can be obtained with 0 risk on the meat packer gateway. Or the free TK and 12 fireaxes that exist on russian gateway. Or the 10 free TC (that spawns next to the cham projector but you chose to not mention) on spider gateway, that you get by cheesing the mobs since it is so easy to enter and do that.

> 3\. If people hate swarmers, then just cut swarmer spawns.

What are you even on about, this PR doesn't touch swarmers at all.

> this feels like another "remove without replacing" case

No content is actually being removed here though, bar the beach (but calling that "content" is a bit of a stretch anyway). It's just being changed so it is accessed differently.

> I just don't see the need to remove a functional and frequently interesting, engaging thing without a viable means of replacing it.

Again with this absolute nonsense, nothing is being removed here apart from the beach. 


I'm all for rational discussion on this topic, _especially_ given it's controversial nature, but I am not engaging in discussion with someone who's making baseless claims and fabricating facts to try and prove a point.

**Comment by AffectedArc07** [comment]
To the "this will make space ruins more packed" people

```diff
 # Minimum number of extra zlevels to generate and fill with ruins
-minimum_zlevels = 2
+minimum_zlevels = 4
 # Maximum number of extra zlevels to generate and fill with ruins
-maximum_zlevels = 4
+maximum_zlevels = 6
```

Its almost like things are configurable for a reason

**Comment by Ordosian** [comment]
> Nobody is complaining about a single cham projector. People are greatly concerned about things like 6 free turret guns that can be obtained with 0 risk on the meat packer gateway. Or the free TK and 12 fireaxes that exist on russian gateway. Or the 10 free TC (that spawns next to the cham projector but you chose to not mention) on spider gateway, that you get by cheesing the mobs since it is so easy to enter and do that.

Moving these to Space ruins will do nothing. If the issue is the loot on some of the gateways, then change the loot? I don't see why removing the Gateway will fix these issues. The only thing this will do is just delay the inevitable.

Gateway is very unique in the way it exists. Only one of these gateways can exist per round. Meaning there is no loot from multiple gateways which could be an issue with space ruins if that is possible, unsure if you changed it.

---

Spider Gateway is the least cheeseable gateway of them all and the hardest gateway of them all. Either way, the 10 TC is useless to a normal player, what are they gonna do with it? The TC has to land in an agent's hands. So that means the agent has to go through the gateway without dying.

Spider Gateway Loot: E-Dagger, Chameleon Projector, Poison Bottle, 10 TC, No-Slip Chameleon Shoes, Safety Override Module, Odysseus, Random High-Level Research Note, Bluespace Power Cell and Retro Laser Gun.

All the above loot except the retro laser gun have to have a 'boss' killed. The Prince of Terror for the Odysseus, Research Note and Bluespace Power cell. For the rest of the loot, you have to kill the Queen of Terror.

---

Russian Gateway is very cheeseable but that is only due to the existence of TK on the map which can be obtained by just running past the guard. Otherwise, the gateway is hard to cheese. Russian Gateway otherwise is fine as long as the Secgear and TK get removed from it/replaced.

Russian Gateway Loot: 3 Secgear lockers, 12 Fireaxes, TK, Mindshield Implanter (Five Charges)

---

Meat Packers Gateway is cheeseable, yes. But the only part of it being cheesed are the turrets. And the only reason this is possible is due to the Ship being made of Silver Walls. These reflect the lasers back at the turrets. These energy guns are uncharged and to get Six is very lucky. Each time you scavenge a turret with a crowbar, there is a chance you get nothing.

Meat Packers Gateway Loot: Syndicate Mask, Suspicious Toolbox + Combat Gloves, Space Suit, ChemMaster, Chem Dispenser, Six Laser Guns (RNG)

---

Moon Outpost 19 (Xenos) Gateway is not really cheeseable. The gain any loot from this gateway, you have to kill the Xeno hive which contains 6 Drone/Hunter Xenos, 2 Sentinels and the Queen. It is just a unique gateway to encounter Xenos as this is the only way outside of Admemery. 

Moon Outpost 19 Gateway Loot: Stun Baton (Charged), Stechkin, Night Vision goggles, Double Barrel shotgun, 2 Insulated, 1 Stechkin Magazine.

A stun baton is encountered in the hive but outside the Queen Chamber. Night Vision goggles and the Stechkin are both encountered in the Queen Chamber. Stechkin Magazine can be found in the Syndicate outpost past the Xeno hive in a locked closet.

---

Beach Gateway is just a relaxing spot for people to go into. It does contain loot but that is not the main point of the gateway and getting to the loot is quite difficult for the average player. The loot is guarded by 3 Melee Pirates and 6 Ranged Pirates. These pirates do a lot of damage so it is not easy to get the loot.

Another place for loot in the Beach Gateway is in the underwater cult hideout. However, none of these cult items does anything, even the forges and altars do nothing for other cultists, merely aesthetic. The Crab render is the only 'usable' loot from the Cult hideout and all it does is open a tear in reality which spawns crabs. It is one-time use.

Beach Gateway Loot: 2 Retro Laser Guns, Boat in a Bottle, Esword (RNG), Pirate Clothing, Arcane Cult Tome, Crab Render

The beach may not be 'content' but it provides something different from all the other gateways. It provides a relaxing place for other crew to relax which none of the other gateways does.

---

So yes, there is a bunch of loot in the gateways. But the loot that has the most value (Xeno + Terror) is guarded by mobs that have a lot of health and do a lot of damage. Terror gateway is the only gateway that gives you weapons to fight the terrors, for Xenos, you have to get it from an outside source.

Russian Gateway has valuable loot in terms of TK and Secgear but those should be removed. No one would argue against it.

However, if the loot is the issue, simply change the loot. I do not see a point in removing a unique exploration tool just to move the gateways into space ruins.

The only time I would be alright with these becoming space ruins is if there was a feasible way of travelling space. For example Space-Pods. Because currently, half of these ruins, most of the player base won't ever take part in. Space navigation/exploration is annoying as stated before and you just float and hope you find something which is not fun or engaging at all. Meanwhile, in a space pod, you could actually travel around space and have a place to store items and a 'base' sort of.

And for the argument that you can just use a jetpack. It is honestly not the same...

**Comment by Norwes1** [comment]
> Norwest I think some of your arguments are valid, especially the education one (at least when older players sdon't just rush it for the loot).
> 
> But I think you make some really bad takes and false claims to try and make your point:
> 
> > 2. The Gateway provides a z-level's worth of Stuff To Do  ...  Losing the Gateway means fewer Theta Station opportunities, because it'll spawn considerably less often
> 
> > This is just not true, gateway levels take up at MOST 1 quarter of a z-level. They are all about the size of the russian ringed space ruin. (The puzzle one). They just seem bigger because there is no frame of reference. So not only are we not "losing an entire z-level's worth of stuff to do" because of size, we also aren't losing it at all, since it's still hanging about. There are so many z-levels that get populated with ruins, the likelyhood of it spawning instead of x or y ruin is slim, a lot of space ruins are duplicated. Wild clames of "theta spawning considerable less often" is just alarmist rhetoric with no statistical evidence. Do not make claims like this without actually doing a controlled test.

While I'm willing to wait and see it gamed out to confirm, I don't see how shifting more content from a separate zone to the Gateway won't result in fewer spawns of other space ruins. The solutions presented here, including adding more z-levels in space, seem problematic given how confusing z-levels already are to navigate (explaining "corner navigation" to new players is enough to make one's head hurt). To get into the specifics:

The standard z-level of 250 by 250 tiles, minus 10 for the borders, means 57,600 usable tiles (240*240). Using the xeno Gateway as a comparison point, as that's a smaller one compared to the Beach or Tspider Gateways, we have a 112x112 space. With another 10 tiles for a border on all sides to avoid conflict with other ruins, we have 14,800 tiles occupied, or roughly 1/4 of a z-level. That ofc assumes perfect efficiency, with no 'spare' tiles of empty space on any side, and borders averaging 20 or 30 tiles of empty space seem far more likely in practice (leading to ~17,500 or ~20,000 tiles occupied).

Taking the current round (32079) as a point of comparison, I'm seeing every space ruin on the log spawning in z-levels 5 and 6. This includes the 'mandatory' spawns of the Syndie spacebase and supply base, the NT medical ship, golem shuttle destination, etc. Theta Station has not spawned, which is fairly unsurprising given its size; there's just not much room. While this is only a single sample, it's as random as can be, and I already see a problem of larger items (Theta) not having room to spawn. Gateway levels would either have to be merged into this as 'random' spawns (leading to a low spawn likelihood given their sizes), as a single 'mandatory' spawn (thereby displacing other items in an already-crowded environment), or by adding more z-levels (and further complicating the already-troubling space navigation).

I'm willing to wait and see, but I see moving the Gateway levels to space as problematic for...well, space itself.

> > 2. Better to keep people entertained going out gunning down NPCs, then ending up in situations that might require a bwoink.
> 
> This is just bonkers, players aren't going to go start breaking server rules because the gateway content requires some effort to run. This is more alarmist rhetoric with no basis in reality.

Please note that I'm not claiming the end of the world here, merely a minor issue that I've seen repeatedly crop up. The particular situation I'm referring to there is where Explorers finish finding the various space ruins, and go digging into the Syndicate researcher base because they don't have anything else to do. While not strictly against the rules, it's obviously not something we want to see, and admins sometimes choose to get involved ICly or OOCly.

This is no different from the eternal Lavaland issue, namely bored Miners using all their absurd amounts of gear to cause trouble (validhunting, greytiding, etc.) on-station. While we obviously bwoink the more obvious cases, unless we're going to remove space exploration entirely, then logically we should try to keep a solid amount of Stuff To Do in order to keep those same players busy. Megafauna and the Syndicate depot are an excellent response to this IMO by providing fun rewards that require significant time and effort to complete, and I see the tspider Gateway as a similar "high investment" area for players.

I'm not trying to be alarmist about some "end of the world" issue, merely pointing out a minor recurring problem which I feel would be worsened with the Gateway being taken away.

> > 3. but complaining about a chameleon projector from a Gateway mission is pretty absurd
> 
> Nobody is complaining about a single cham projector. People are greatly concerned about things like 6 free turret guns that can be obtained with 0 risk on the meat packer gateway. Or the free TK and 12 fireaxes that exist on russian gateway. Or the 10 free TC (that spawns next to the cham projector but you chose to not mention) on spider gateway, that you get by cheesing the mobs since it is so easy to enter and do that.
> 
> > 3. If people hate swarmers, then just cut swarmer spawns.
> 
> What are you even on about, this PR doesn't touch swarmers at all.

I apologize re, any confusion about swarmers, as I was told that exasperation with swarmers was helping lead to this PR. I've no dispute whatsoever with removing the TK injector and Sec loot from the Russian Gateway, or the turret guns from the meatpacker one, if you find those to be recurring problems; personally, I'd cheer on removing the TK one in particular. If people want to tweak the older Gateways, I'm all ears.

I don't see the tspider Gateway loot as an issue, given that it involves fighting one's way through a base's worth of NPCs, and the 10TC gained are useless without an uplink (which is therefore a reasonable reward for the Syndie who spends the significant time fighting their way through the place). It's a useful reward, but comes at a commensurate cost in time, effort, and risk expended, and the primary reward itself (the TC) requires a Syndicate uplink to utilize. The tspider, underwater/pirate beach, and xeno Gateways all appear to have reasonable risk-reward ratios, with the meatpacker and USSP ones being problematic notably in their 'background' gear (the turret guns, fireaxes, and Security gear). Those issues can be fixed with a quick revision, though, and therefore aren't necessarily indicative of a fundamental problem in play.

> > this feels like another "remove without replacing" case
> 
> No content is actually being removed here though, bar the beach (but calling that "content" is a bit of a stretch anyway). It's just being changed so it is accessed differently.
> 
> > I just don't see the need to remove a functional and frequently interesting, engaging thing without a viable means of replacing it.
> 
> Again with this absolute nonsense, nothing is being removed here apart from the beach.
> 
> I'm all for rational discussion on this topic, _especially_ given it's controversial nature, but I am not engaging in discussion with someone who's making baseless claims and fabricating facts to try and prove a point.

I apologize if I'm giving a hostile vibe here, as I merely wanted to re-state that I don't think losing the Gateway and some of its content (the beach) would be a net positive. I understand you're putting in significant work for this PR, and I respect that effort spent even if I disagree with the end result. That being said, I felt my initial response was a bit too noncommittal, and wanted to clarify that I really don't think this is a good idea, especially before a lot of effort was put on the end result.

Aside from what I've mentioned above, I see two issues with the claim that "nothing is being removed." The first claim, namely that the content "isn't being replaced," is that the content is being rendered far more difficult to reach. It's still accessible to the hardcore Explorers who can navigate z-levels like the backs of their hands, but I'm far more interested in making the content accessible to players with <100 hrs experience. The Gateway provides that by being an easy 'port away, and is equally accessible for admin events. Without the Gateway, we're left with content that will rarely be seen by the bald-ier players we _want_ to reach it. This could be partially accommodated by giving Explorers access to a single-use teleporter which'd send them to the xeno Gateway or a similar place...but at that point, why remove the Gateway in the first place?

Similarly, easy access is a key issue when setting up off-station events. Sometimes that's desirable, like when running "space dwarfs" on Lavaland, but by and large we want to make players get in and out easily, no? Teleporters are possible, but problematic due to the likelihood of people messing with 'em (deconstructing the teleporter, messing with the destination, etc. - I've had multiple issues of that same type when running off-station events before). This is problematic given how desirable off-station events are on the whole: they're entirely voluntary to participate in unlike on-station events, they allow for players to "cut loose" with techniques and equipment that's generally prohibited on-station, while safe and effective transport means that they can still involve on-station departments and players (wounded players going to Medbay, requiring gear from Cargo and Science, etc). Additionally, ICly it's difficult to justify setting up a purpose-built teleporter, whereas the Gateway provides an easy free-standing justification for why and how there's a permanent gate leading to a Russian station or tspider-infested outpost. Without the Gateway, events like Denth's most recent "Syndicate station" on the Russian Gateway are made considerably more difficult to run. 

The alternative there is teleporters (with the various issues of disruption, as mentioned above; even walling off the teleporter console and station doesn't stop the hub from being messed with, such as APC Overload events). Shuttles aren't really feasible, as they require a high degree of knowledge and can still easily go wrong if mismanaged even slightly; I've seen a 'min accidently wipe out the Bridge on the emergency shuttle with a custom shuttle. The Gateway is simple, indestructible, and requires almost no labor whatsoever. It's the best option for events like that, and I think it'd be a shame to lose it.

Secondly, re your claim that the Beach is "content-less:" while I believe Ordosian has already covered the question of the pirates, I'd also argue that the underwater exploration level there is a perfect starter area for would-be Explorers:
1. It's large enough to get lost in, but still navigable without having to understand weird mechanics like the SS13 GPS. People can, and do, still navigate it with simple techniques like inertial navigation (i.e. "take twenty steps north from the anchor, drop a flashlight, then retrace my steps afterwards").
2. It's a dangerous environment (requiring internals), but doesn't also require an understanding of gravity-less movement, vacuum suits, jetpacks, etc. to get through. People won't die underwater from their jetpack running out of juice - and given that EVA only starts with a few 'packs, it also doesn't provide the same "gatekeeping" effect that the lack of jetpacks does to space Explorers.
3. The beach underwater area is hostile (with sea carp), but isn't threatening enough to absolutely murder baldies by the boatload. There's still some deaths down there, but compared to the casualties of newbies trying to traverse the Syndicate depot, it's a breeze.
4. It's wonderfully atmospheric, with the combination of darkness and water noises providing a uniquely oppressive environment that's not easily replicated elsewhere.

Yes, the beach area is quickly traversed by experienced players, but for newbie players it's an excellent learning opportunity. I've also run plenty of off-station events on the Beach area itself, whether a Syndicate diplomatic summit to a 'borg-transformed former lifeguard, and think it'd be a shame to lose it.



The primary issue you've raised above, namely of certain Gateways having some problematic loot, is easily rectified by simply removing said loot. I certainly won't shed any tears if the TK injectors and Sec gear in the Russian Gateway goes away, and if it saves the Gateways, I'd be happy to work on design documents for the Gateways deemed in need of an overhaul. However, I really do think that having a ready-made teleporter to an off-station place is a useful asset, that it provides an accessible means for people to learn how2explore, and that removing it would be a net loss for the server gameplay as a whole.

**Comment by S34NW** [comment]
This PR as it is will not be carried on, despite design team approval.

Watch this space.

**Comment by S34NW** [comment]
(Also norwest no aggro, sorry if I came on too hard with my response to your feedback)

**Pull request closed without merging**